x-pack/plugin/otel-data/src/main/resources/component-templates/ecs-tsdb@mappings.yaml

```diff
@@ -0,0 +1,20 @@
+version: ${xpack.oteldata.template.version}
+_meta:
+  description: Default mappings for the logs index template installed by x-pack
+  managed: true
+template:
+  mappings:
+    dynamic_templates:
+      - ecs_ip:
+          mapping:
+            type: ip
+          path_match:
+            - "ip"
+            - "*.ip"
+            - "*_ip"
+          match_mapping_type: string
```

Comment:
Because of https://github.com/elastic/elasticsearch/issues/110387, we can't map all `*.ip` fields to the  `ip` field type.

```suggestion
```